js/track.js

```diff
@@ -180,7 +177,7 @@ export class TrackManager {
     }
     this._tracks.splice(index, 1);
 
-    track.channel.floatArray.fill(0);
+    track.channel.getFloatArray().fill(0);
```

Comment:
This might not be sufficient on firefox.

```suggestion
    const floatArray = track.channel.getFloatArray();
    floatArray.fill(0);
    track.channel.setFloatArray(floatArray);

```